### PR TITLE
Update universite-du-quebec-a-montreal.csl

### PR DESCRIPTION
**Pull request by rmzelle**
Per https://forums.zotero.org/discussion/63923/make-a-non-listed-csl-1-0-1-valid

### CODE REVIEW

File: universite-du-quebec-a-montreal.csl [hunk 1]
```diff
@@ -1,487 +1,542 @@
...
-                <name and="text" delimiter=", "/>
-              </names>
-            </if>
-          </choose>
-        </group>
+      <if type="chapter paper-conference entry-encyclopedia entry-dictionary" match="any">
+        <names variable="editor container-author" delimiter=", " suffix=", ">
+          <name and="text" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=" (" text-case="title" suffix=")"/>
+          <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
```
Comment (rmzelle) [inline universite-du-quebec-a-montreal.csl]: These double labels are weird. I'm rather surprised it even validates.

(csl.rnc uses:

```
    element cs:names {
      names.attributes,
      ((names.name?, names.et-al?) & names.label?),
      names.substitute?
    }
```

which I would think would only allow a single `<label/>`)

File: universite-du-quebec-a-montreal.csl [hunk 2]
```diff
@@ -1,487 +1,542 @@
...
-  <macro name="anon">
+  <macro name="secondary-contributors">
     <choose>
-      <if variable="author editor translator" match="none">
-        <text term="anonymous" form="short" text-case="capitalize-first" suffix="." strip-periods="false"/>
+      <if type="article-journal book chapter paper-conference entry-encyclopedia" match="none">
+        <names variable="editor" delimiter=", " prefix=" (" suffix=")">
+          <name name-as-sort-order="all" and="text" sort-separator=", " initialize-with=". " delimiter=", " delimiter-precedes-last="never"/>
+          <label form="short" prefix=", " text-case="title"/>
+          <label form="short" prefix=" (" suffix=")" text-case="lowercase"/>
```
Comment (rmzelle) [inline universite-du-quebec-a-montreal.csl]: Idem.

File: universite-du-quebec-a-montreal.csl (hunk 1 above)
Comment (rmzelle) [inline universite-du-quebec-a-montreal.csl]: Nevermind. I copied that schema fragment from "master", where we already fixed this post-1.0.1: https://github.com/citation-style-language/schema/commit/850a86fb56f10c198ef051159df07386d19dd56a

Comment (rmzelle) [inline universite-du-quebec-a-montreal.csl]: Should still fix the style, though.

Comment (rmzelle) [inline universite-du-quebec-a-montreal.csl]: @damnation333, do you have a time to check which of these two labels we should keep here and below?

File: universite-du-quebec-a-montreal.csl (hunk 2 above)
Comment (adam3smith) [inline universite-du-quebec-a-montreal.csl]: sorry, I didn't see the ping. The lowercase labels in parentheses are correct both here and above

File: universite-du-quebec-a-montreal.csl (hunk 1 above)
Comment (rmzelle) [inline universite-du-quebec-a-montreal.csl]: (@damnation333, never mind, @adam3smith already helped out: https://github.com/citation-style-language/styles/pull/2458#discussion_r99272512)

**Pull request merged**